codec/src/main/java/io/netty/handler/codec/compression/Bzip2BlockCompressor.java

```diff
@@ -113,6 +115,7 @@ private void writeSymbolMap(ByteBuf out) {
             for (int j = 0, k = i << 4; j < HUFFMAN_SYMBOL_RANGE_SIZE; j++, k++) {
                 if (blockValuesPresent[k]) {
                     condensedInUse[i] = true;
+                    break;
```

Comment:
Are you sure this is correct ? /cc @idelpivnitskiy

Reply:
I agree, this looks very suspicious.

Reply:
Ufff, it's been years... I would be hesitant to make this change without justifying test that can demonstrate incorrect behavior without `break`.